src/assets/sitemapper.js

```diff
@@ -47,23 +55,35 @@ export default class Sitemapper {
    *  .then((sites) => console.log(sites));
    */
   async fetch(url = this.url) {
-    let sites = [];
+    let results = {
+      url: '',
+      sites: [],
+      errors: []
+    };
     try {
       // crawl the URL
-      sites = await this.crawl(url);
+      results = await this.crawl(url);
     } catch (e) {
       if (this.debug) {
         console.error(e);
       }
     }
 
     // If we run into an error, don't throw, but instead return an empty array
-    return {
-      url,
-      sites,
-    };
-  }
+    if (!this.returnErrors) {
+      return {
+        url,
+        sites: results.sites || []
+      };
+    } else {
+      return {
+        url,
+        sites: results.sites || [],
+        errors: results.error || []
+      };
+    }
```

Comment:
@bsq-panagiotis Actually never mind, the signature on Sitemapper.fetch is the same just an added property, so it should be okay. I was wondering why the tests were still passing. Can keep it to a minor version bump.

Reply:
I believe the signature should stay the same though instead of showing `errors` only if there is a `returnErrors` option passed in.
We can remove the returnErrors option, and just always return the errors property. The user can decide to use it or not.

Reply:
@seantomburke  Thanks for taking the time to review the changes. 
I agree on your comment for the `returnErrors` option, I only included it to reduce the impact of the changes on existing setups, but I don't think this change can break anything.